test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -286,4 +286,19 @@ def test_successful_validate_pinless_debit_card
     assert_equal 'Y', response.params["status"]
     assert_equal true,  response.success?
   end
+
+  def test_network_tokenization_authorize_and_capture
```

Comment:
This is a little unclear, I have a feeling that some gateways may do and auth+capture on their end in this case but in general yeah it will just be declined.

I'm on the fence about whether we should handle this or let the gateway handle it cause most integrations that use AM would handle the decline properly but if we started to introduce another exception type it might cause some problems (not really an excuse I guess either)